MonoGame.Framework/Color.cs

```diff
@@ -296,127 +308,98 @@ public Color(float r, float g, float b, float alpha)
             A = (byte)MathHelper.Clamp(alpha * 255, Byte.MinValue, Byte.MaxValue);
         }
 
+        #endregion
+
+        #region Public Properties
+
         /// <summary>
-        /// Gets or sets the blue component.
+        /// The alpha component of this color.
         /// </summary>
         [DataMember]
-        public byte B
+        public byte A
         {
             get
             {
                 unchecked
                 {
-                    return (byte) (this._packedValue >> 16);
+                    return (byte)(this._packedValue >> 24);
                 }
             }
             set
             {
-                this._packedValue = (this._packedValue & 0xff00ffff) | ((uint)value << 16);
+                this._packedValue = (this._packedValue & 0x00ffffff) | ((uint)value << 24);

```

Comment:
This is why we avoid re-sorting code in a file. I have to now check that you didn't change functionality here because at first glance it looks like you changed how packing works.

It makes this diff much more difficult to understand. You should avoid it in the future.